NOTICE

```diff
@@ -0,0 +1,78 @@
+Apache Cassandra Apache Spark Connector
+Copyright 2014 The Apache Software Foundation
```

Comment:
Should the copyright year be `2025` here?

Reply:
No.  Copyright is always the year of origin (creation), and here we are transferring that existing copyright from DataStax to the ASF.